src/main/java/ch/njol/skript/expressions/ExprTime.java

```diff
@@ -76,56 +64,65 @@ public Class<?>[] acceptChange(final ChangeMode mode) {
 		switch (mode) {
 			case ADD:
 			case REMOVE:
-				return CollectionUtils.array(Timespan.class);
+				// allow time to avoid conversion to timespan, which causes all sorts of headaches
+				return CollectionUtils.array(Time.class, Timespan.class);
 			case SET:
 				return CollectionUtils.array(Time.class, Timeperiod.class);
```

Comment:
Not too sure how important the note is, but we can likely rely on a fallback to the case SET return so we don't have two of the same duplicates

Reply:
i don't think it really matters, also the note should probably stay to help anyone who dares touching this code